index.html

```diff
@@ -4,11 +4,138 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css" />
+    <link rel="stylesheet" href="https://innotion-ui-library.netlify.app/in-notion.css">
     <link rel="stylesheet" href="global.css">
-    <script src="app.js"></script>
+    <script src="app.js" defer></script>
     <title>inNotion Quiz</title>
 </head>
-<body>
-    
-</body>
+<body class="content">
+    <div class="header header-grid fixed-header row-flex">
+      <nav class="navbar row-flex">
+        <a class="m-l-3" href="/"><h3 id="logo">in.notion <span class="text-shd">Quiz</span></h3></a>
+        <ul class="nav-items row-flex no-bullet">
+          <li class="btn nav-link-btn h3 m-v-2 m-r-3"><a href="#">Leaderboard</a></li>
+          <li class="btn nav-link-btn h3 m-v-2 m-r-3"><a href="#">Rules</a></li>
+        </ul>
+        
+        <ul class="row-flex no-bullet">
+            <li class="dark-mode icon-toggle icon-btn rd-bdr grid-ctr colored-text m-r-3"></li>
+            <li class="nav-icon-btn icon-btn rd-bdr grid-ctr wt-text m-r-3">
+                <a href="#" class="grid-ctr">
```

Comment:
Use `javascript:;` in place of `#` it will not make your click go to top.

Reply:
made the changes.